.goreleaser.yml

```diff
@@ -0,0 +1,24 @@
+builds:
+  - binary: grpcurl
+    main: ./cmd/grpcurl
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - 386
+    ldflags:
+      - -s -w -X main.version=v{{.Version}}
```

Comment:
I copied this line from the documentation, and the `-s -w` are part of the default (see below). The main difference I made was to drop the fields we don't use and to add a `v` at the beginning of the version. I did the prefix here and not in the code because the code would print out `v` in front of the `dev build...` string which wouldn't look great.

The default value is:
```
-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
```
https://goreleaser.com/customization/#Name%20Templates